src/transport/SessionManager.h

```diff
@@ -349,6 +349,17 @@ class DLL_EXPORT SessionManager : public TransportMgrDelegate, public FabricTabl
 
     void ExpireAllPASESessions();
 
+    /**
+     * @brief
+     *   Marks all active session that matches provided arguments as defunct.
+     *
+     * @param node    Scoped node ID of the active sessions we should mark as defunct.
+     * @param type    Type of session we are looking to mark as defunct. If matching
+     *                against all types of sessions is desired, NullOptional should
+     *                be passed into type.
+     */
+    bool MarkSessionsAsDefunct(const ScopedNodeId & node, const Optional<Transport::SecureSession::Type> & type);
```

Comment:
This needs to document the return value.  Or return void, since no one uses the return value....

Reply:
This will return void (and possibly this function will be removed altogether) in https://github.com/project-chip/connectedhomeip/issues/20452, but right now there is something use the returned value.